mobile/examples/phi-3/android/app/src/main/java/ai/onnxruntime/genai/demo/MainActivity.java

```diff
@@ -101,99 +160,98 @@ public void run() {
 
     @Override
     protected void onDestroy() {
-        try {
-            genAIWrapper.close();
-        } catch (Exception e) {
-            Log.e(TAG, "exception from closing genAIWrapper", e);
-        }
-        genAIWrapper = null;
+        tokenizer.close();
+        tokenizer = null;
+        model.close();
+        model = null;
```

Comment:
nit: is there any need to set to null if onDestory is the last point the instance is usable?

Reply:
not sure. did this to make sure for garbage collection.